docs/terminal/eastsale.md

```diff
@@ -1,10 +1,247 @@
----
-  title: East Sale TCU
----
+# East Sale TCU
 
---8<-- "includes/abbreviations.md"
+## Positions
 
-Reserved.
+| Name               | ID      | Callsign             | Frequency        | Login Identifier    |
+| ------------------ | ------- | ------------------- | ---------------- | ------------------- |
+| **East Sale TMA**  | **ESL** | **East Sale Approach** | **123.300**      | **ES_APP**          |
 
-!!! note
-    Additional charts to the AIP may be found in the RAAF TERMA document, available towards the bottom of [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
\ No newline at end of file
+## Airspace
+
+ES TCU controls the airspace within R360, from `SFC` to `F210`. The TCU controller has control from the surface up; if circuit operations need to be handled by East Sale Tower, a release is required.
```

Comment:
Perhaps just a slight tweak to the wording. Need to make sure it is clear that Enroute own the underlying Class G, but ES TCU owns R360A from the SFC.

Reply:
Reworded it, I've mentioned the normal configuration of R360B-F being active. That what seems to be happening when I go there.

How's that sound?